src/index.js

```diff
@@ -1,7 +1,8 @@
 // Copyright 2018 Roger Meier <roger@bufferoverflow.ch>
 // SPDX-License-Identifier: MIT
 
-var httperror = require('http-errors');
+import Gitlab from 'gitlab';
+import {httperror} from 'http-errors';
 
 function Auth(config, stuff) {
   var self = Object.create(Auth.prototype);
```

Comment:
`verdaccio v3` already support ES6 classes. It's your decision do `Object.create(Auth.prototype);` or just do eg: https://github.com/verdaccio/verdaccio-htpasswd/blob/master/src/index.js#L9